lib/ansible/modules/packaging/os/homebrew_cask.py

```diff
@@ -472,6 +484,24 @@ def _run(self):
         self.message = "You must select a cask to install."
         raise HomebrewCaskException(self.message)
 
+    # sudo_password fix ---------------------- {{{
+    def _run_command_with_sudo_password(self, cmd):
+        sudo_askpass_file = tempfile.NamedTemporaryFile()
```

Comment:
Use a context manager when dealing with files.
```suggestion
        with tempfile.NamedTemporaryFile() as sudo_askpass_file:
```